echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/monitor/DockerEventMonitor.java

```diff
@@ -77,7 +80,19 @@ protected boolean isSuccessfulTriggerEvent(final TriggerEvent event) {
   @Override
   protected Function<Trigger, Pipeline> buildTrigger(Pipeline pipeline, TriggerEvent event) {
     DockerEvent dockerEvent = (DockerEvent) event;
-    return trigger -> pipeline.withTrigger(trigger.atTag(dockerEvent.getContent().getTag()));
+    DockerEvent.Content content = dockerEvent.getContent();
+
+    String name = content.getRegistry() + "/" + content.getRepository();
+    String reference = name + ":" + content.getTag();
+    List<Artifact> artifacts = Collections.singletonList(Artifact.builder()
+      .type("docker/image")
+      .name(name)
+      .version(content.getTag())
+      .location(reference)
```

Comment:
I don't think "location" has any special meaning for docker images (yet). I would drop this call here.

Reply:
Done.